### DISCUSSION TITLE
Several shapes to the body

### DISCUSSION THREAD

**Discussion by Hromon** [bepu/bepuphysics2#300] [Q&A] [answered]
How is it possible, if possible, to tie several shapes to the body with an offset?
For example, I have a voxel chunk, and I want to generate a body that consists of voxel shapes with an offset.

**Comment by Hromon** [comment]
So did GodotEngine:
![image](https://github.com/bepu/bepuphysics2/assets/29508584/4071eb9e-e954-4fe1-b416-0bce9200e0c0)

**Comment by RossNordby** [comment, marked as answer]
Use a `Compound` or `BigCompound` shape. `BigCompound` has an acceleration structure, so if an acceleration structure is likely to reduce the number of bounding box tests needed compared to a `Compound`, use it instead.

It's also possible to make custom collidables like in the `VoxelCollidableDemo`, though I'd avoid that unless there's a strong reason to do it.
